### ISSUE TITLE
Right Sidebar Text Position

### ISSUE THREAD

**Issue by billy-codes** [dxcore35/Suddha-theme#3]
Amazing work on the theme dxcore,

Just a slight issue I'm facing and it's the right sidebar. 
The text seems to be mispositioned from its surrounding elements.

For instance, in the backlinks section, the text is blocked by the menu bar on top.
I've corrected it by adding a property "padding-top:10px;" under backlinks pane.
The outline pane also has the texts overlapping each other.

<img width="380" alt="Screenshot 2021-02-12 at 7 07 38 PM" src="https://user-images.githubusercontent.com/52131279/107760994-9db5e480-6d65-11eb-9b5e-41b1247adf6a.png">

<img width="332" alt="Screenshot 2021-02-12 at 7 05 30 PM" src="https://user-images.githubusercontent.com/52131279/107760821-562f5880-6d65-11eb-8c3f-d92975e7cb04.png">

I'm using version 1.0 on a Mac OS for now.
Many thanks again for your contribution!

Cheers,
Billy

**Comment by dxcore35** [comment]
I forgot to update my fixed version to Github.
Please test last release: https://github.com/dxcore35/Suddha-theme/releases/tag/1.2
- If it is fixed than feel free to close the issue.
- If you find other bugs, just open new issue I will look into them.

Thanks for appreciation!

**Comment by billy-codes** [comment]
That was quick,

Works perfectly now and looks beautiful like it always is!

Many thanks for the fix!
I'll close this issue now and let you know if there's anything else. 

Cheers,
Billy
